packages/appstore/src/constants/platform-config.ts

```diff
@@ -0,0 +1,43 @@
+import { getPlatformSettingsAppstore, routes } from '@deriv/shared';
+
+import { localize } from '@deriv/translations';
+
+const platform_config = [
```

Comment:
why some of them has `link_to` and some `href` since the behavior is the same?

Reply:
@mahdiyeh-fs Paths that are set on our router configuration uses link_to. href are using for redirecting to other domains like smart trader and binary bot.

Reply:
as long as the behaviour are the same no need to use different keys for that, keep `link_to` for all of them. the below is the part of code that you use these keys, so chnge them as well
![image](https://user-images.githubusercontent.com/82078941/196128892-b4a656d8-0e8a-4881-9f70-4698e95de136.png)

Reply:
@mahdiyeh-fs  The behaviour is not same actually, some url's are redirecting inside the same domain and others are redirecting to a new domain.

Can you please check the use case over there and give a suggestion.?

Reply:
<img width="821" alt="image" src="https://user-images.githubusercontent.com/97999159/196339266-6194e504-2bf3-43b3-af74-e00ecd42ade1.png">
 We are handling same scenario here as well.

Reply:
you're right, that's fine